streams-bootstrap/src/main/java/com/bakdata/kafka/KafkaStreamsApplication.java

```diff
@@ -159,7 +161,9 @@ public void close() {
      * This must be set to a unique value for every application interacting with your kafka cluster to ensure internal
      * state encapsulation. Could be set to: className-inputTopic-outputTopic
      */
-    public abstract String getUniqueAppId();
+    public String getUniqueAppId() {
```

Comment:
If the user doesn't implement this and calls it, they will get `null` returned, correct? 🤔

Reply:
See comments in Slack/new version